lib/triangle_farms/calendar/time.ex

```diff
@@ -0,0 +1,15 @@
+defmodule TriangleFarms.Calendar.Time do
```

Comment:
My motivation for pulling out this module (which may not be the best named) was seeing the "{RFC3339}" string repeated a lot. It also isolates the Timex dependency, so if you ever decide to change time libraries you have your own interface to handle rather than the external.

Reply:
This is great, it'll also make testing easier since I could swap out the Time module for a TestTime module that defines a now function with a deterministic value.